brainscore_language/submission/endpoints.py

```diff
@@ -118,8 +141,12 @@ def parse_args() -> argparse.Namespace:
     args = parse_args()
     args_dict = vars(args)
 
-    if 'user_id' not in args_dict or args_dict['user_id'] == None:
-        user_id = get_user_id(args_dict['author_email'])
-        args_dict['user_id'] = user_id
-    
-    run_scoring(args_dict)
+    if args.fn == 'run_scoring':
+        if 'user_id' not in args_dict or args_dict['user_id'] == None:
+            new_user_id = create_user(args_dict['domain'], args_dict['author_email'])
+            args_dict['user_id'] = new_user_id
```

Comment:
`create_user()` was changed to `get_user_id()`

```suggestion
            user_id = get_user_id(args_dict['author_email'])
            args_dict['user_id'] = user_id
```